packages/lesswrong/components/comments/ModerationGuidelines/ModerationGuidelinesBox.jsx

```diff
@@ -80,11 +83,12 @@ class ModerationGuidelinesBox extends PureComponent {
       Suffix: "... (Read More)",
       Strict: false
     }
-    const userGuidelines = `${document.user ? `<b>${document.user.displayName + "'s moderation guidelines" } </b>: <br>
-    <span class="${classes[moderationStyle]}">${moderationStyleLookup[moderationStyle]}</span>` : ""}
+    const userGuidelines = `${document.user ? `<b>${document.user.displayName + "'s commenting guidelines"}</b>: <span class="${classes[moderationStyle]}">${moderationStyleLookup[moderationStyle]}</span> <br/>` : ""}
     ${document.moderationGuidelinesHtmlBody || ""}`
+
     const combinedGuidelines = `
       ${(document.moderationGuidelinesHtmlBody || moderationStyle) ? userGuidelines : ""}
+      ${document.moderationGuidelinesHtmlBody ? '<hr class="dividerBlock"></hr>' : '</br>'}
```

Comment:
These style changes don't match the PR title, but they look reasonable. Assuming you meant for these to be reviewed, I'm fine with them. (But if these were half-finished make sure you're happy with them first!)

Reply:
I based this off of the other branch that we just merged #1471